pkg/machinery/config/types/runtime/testdata/kmsglog.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1alpha1
+kind: KmsgLogConfig
+name: apiSink
```

Comment:
why do we have a name here?

Reply:
this is a `NamedConfiguration`, i.e. it has a `name:`, so there might be multiple config documents, so you can have kmsg log streaming to multiple destinations